README.md

```diff
@@ -171,6 +171,18 @@ interval between any two periodic sync events can be greater than the requested
 is up to the browser implemetation. Also, the interval between any two periodic sync events for a
 registration doesn't have to be the same.
 
+Since a website will most likely download resources from the network when processing a
+`periodicsync` event, multiple `periodicsync` events can cause a lot of data consumption. Like other
+service worker events, `periodicsync` MAY be subject to the service worker execution timeout. Every
+time the browser dispatches these events, the browser will potentially be kept awake for this
+duration, which is a few minutes. This can cause battery drain. The browser implementation SHOULD
+therefore ensure an appropriate interval for firing these events. It MAY cap this interval per
+origin and across origins. It MAY also enforce limits on how often it is woken up to dispatch these
```

Comment:
"It MAY also....to dispatch these"

I think the premise of this is captured in the "across origins" part of the previous sentence. That already grants implementations the necessary flexibility to have such scheduled wake-ups.

Reply:
Removed.